### DISCUSSION TITLE
Can't Watch Twitch After Updating to 6.0.0-1

### DISCUSSION THREAD

**Discussion by itsotter** [streamlink/streamlink#5473] [Q&A] [answered]
### Description

Hi, I'm a longtime streamlink user and I'm getting this error message when trying to launch Twitch streams after updating:

![attributeerror](https://github.com/streamlink/streamlink/assets/33299517/8091aefc-0be2-48a2-a60a-1ef06c2431a2)

I found a related thread about this AttributeError ( https://github.com/streamlink/streamlink/issues/5468 ) but it's already closed and seemed to be about writing plugins. Does this mean I have to edit the existing twitch plugin to get it to work on the new version? This is over my head as a non-coder. Or am I doing something else wrong?

Thanks for any help!

**Comment by bastimeyer** [comment, marked as answer]
You are either using a custom version of the Twitch plugin
https://streamlink.github.io/cli/plugin-sideloading.html
or an outdated one for some reason. Make sure that everything is up-to-date and that you are not using 3rd party modifications.

**Reply by bastimeyer** [reply]
Your install path is in `C:\Program Files (x86)\`, but you are **sideloading** an outdated version of the Twitch plugin from `%APPDATA%\streamlink\plugins`.

**Comment by itsotter** [comment]
Thanks so much! I use streamlink every day so I was seriously worried and I appreciate you taking the time to set me straight.

To make things easier for any other people as dumb as me: I reinstalled from 
streamlink-6.0.0-1-py311-x86_64.exe to fix the apparent path inconsistency and downloaded a fresh twitch.py from https://github.com/streamlink/streamlink/blob/master/src/streamlink/plugins/twitch.py to put in my appdata folder. Now it's all good again.

**Reply by bastimeyer** [reply]
> downloaded a fresh twitch.py from [master/src/streamlink/plugins/twitch.py](https://github.com/streamlink/streamlink/blob/master/src/streamlink/plugins/twitch.py) to put in my appdata folder

Why are you even sideloading?

**Reply by itsotter** [reply]
Honestly, I didn't know I was and I don't know how to stop doing so... I am just barely savvy enough to put my mpv path in the cfg file. I don't see anything about this in that cfg file looking over it now, so I don't know how I managed to mess this up.

**Reply by bastimeyer** [reply]
I have already linked you the docs which explain what sideloading is.
https://streamlink.github.io/cli/plugin-sideloading.html

If you don't understand what you're doing, then you probably shouldn't be doing it in the first place.

Just delete the file, so that Streamlink will use the bundled plugin version and not some custom one.

**Reply by itsotter** [reply]
I get it now and I've deleted it. Thanks again for your patience.
